### PR TITLE
Maintenance update

### PR DESCRIPTION
**Pull request by threema-lenny**
Supersedes #66
Includes #67
Supersedes #68

### CODE REVIEW

**Comment by threema-lenny** [comment]
Once merged, I'll follow up with a release. :slightly_smiling_face:

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,7 +1,15 @@
 Changelog
 *********
 
+`8.0.0`_ (2024-08-21)
+---------------------
+
+- Add support for Python 3.10 and Python 3.11
```
Comment (threema-eduard) [inline CHANGELOG.rst]: what about Python 3.12? 3.10 was already supported AFAIK.

Comment (threema-eduard) [reply CHANGELOG.rst]: And shouldn't the new "GATEWAY_API_URL" env option be mentioned?

**Review by threema-eduard — CHANGES_REQUESTED** [review]

**Comment by threema-eduard** [comment]
LGTM apart from the little comments above!

File: threema/gateway/bin/gateway_client.py [hunk 2]
```diff
@@ -421,6 +421,7 @@ async def send_video(ctx, **arguments):
 @click.argument('secret')
 @click.argument('private_key')
 @click.argument('file_path')
+@click.argument('caption', required=False)
```
Comment (threema-lenny) [inline threema/gateway/bin/gateway_client.py]: Good addition. But I think I'd prefer a `@click.option` here. :slightly_smiling_face:

Comment (threema-eduard) [reply threema/gateway/bin/gateway_client.py]: 1 sec

File: CHANGELOG.rst (hunk 1 above)
Comment (threema-lenny) [reply CHANGELOG.rst]: Fixed!

**Review by threema-eduard — APPROVED** [review]

**Pull request merged**